db/migrate/20160324124129_add_index.rb

```diff
@@ -0,0 +1,4 @@
+class AddIndex < ActiveRecord::Migration
+  def change
```

Comment:
I think that would do it. Or you can manually change the one line that changed, or you can `git checkout master db/schema.rb`.